dependencies.py

```diff
@@ -82,7 +83,11 @@ def ensure_everything_installed():
         missing += _check_presence(WINDOWS_PLATFORM_DEPENDENCIES)
         _ensure_h5py_installed()
     target_dir = _dependencies_target_dir()
-    _install_dependencies(missing, target_dir=target_dir)
+
+    if len(missing) > 0:
```

Comment:
`if missing` is ook afdoende. Een lege lijst geldt automatisch als False. Je hoeft geen `len()` en geen `> 0` te doen.